.changeset/little-insects-pull.md

```diff
@@ -0,0 +1,7 @@
+---
+"@marigold/docs": minor
+"@marigold/components": minor
+"@marigold/system": minor
+---
+
+feat: Introduce `Template` component
```

Comment:
Why not _Introduce `Grid`component_? It's kinda confusing we call it everywhere Grid component, also in the docs.

Reply:
yes this was me when I was not sure how to call it :D